oci/oci.go

```diff
@@ -579,7 +579,7 @@ func (r *Runtime) UpdateStatus(c *Container) error {
 	}
 
 	if c.state.Status == ContainerStateStopped {
-		exitFilePath := filepath.Join(c.bundlePath, "exit")
+		exitFilePath := filepath.Join("/var/run/crio/exits", c.id)
```

Comment:
Can you make this a constant somewhere so that it can be shared between functions?